src/formatDate.js

```diff
@@ -50,7 +50,36 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const mainDate = date.split(fromFormat[fromFormat.length - 1]);
+  const objDate = {};
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    objDate[fromFormat[i]] = mainDate[i];
+
+    if (fromFormat[i] === 'YY' && mainDate[i] >= 30) {
```

Comment:
Use template strings instead of string concatenation. Replace '19' + mainDate[i] with `19${mainDate[i]}`.